### PR TITLE
macos-vpn-server: Submission of macOS-native VPN (L2TP-IPSec-PSK) Server

### PR DESCRIPTION
**Pull request by essandess**
macos-vpn-server: Submission of macOS-native VPN (L2TP-IPSec-PSK) Server

* Configures native VPN Server on macOS
* Based on deprecated VPN Server in macOS Server.app
* MacPorts-specific launchdaemon for vpnd, Keychain PSK
* Reference: https://developer.apple.com/support/macos-server/macOS-Server-Service-Migration-Guide.pdf

###### Type(s)

- [x] submission
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.5 18F132
Xcode 10.2.1 10E1001 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by essandess** [comment]
May I please request some Buildbots help?

I believe these errors arise from catching some bash non-0 exit code, but don't see the specific line in the logs.

One workaround is to append `|| /usr/bin/true` to all shell commands within the Portfile, but I'd rather see the actual issue.

My local box works end-to-end with this submission.

Here's the Azure paste: https://paste.z0k.xyz/c01a6b4609d9

File: net/macos-vpn-server/Portfile [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+PortSystem              1.0
+
+name                    macos-vpn-server
+# use the latest macOS version as the version number of this port
+version                 10.14.5
+homepage                https://opensource.apple.com/source/ppp/ppp-838/Helpers/vpnd/vpnd.5.auto.html
+categories              net
+platforms               darwin
+supported_archs         noarch
+maintainers             @essandess
```
Comment (mf2k) [inline net/macos-vpn-server/Portfile]: Can you add your email and `openmaintainer` here?

Comment (essandess) [reply net/macos-vpn-server/Portfile]: Done, thanks.

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**